addons/spreadsheet/static/src/chart/odoo_chart/odoo_line_chart.js

```diff
@@ -121,5 +132,8 @@ function getLineConfiguration(chart, labels) {
     if (chart.stacked) {
         config.options.scales.yAxes[0].stacked = true;
     }
+    if (chart.cumulative) {
+        config.options.scales.yAxes[0].cumulative = true;
```

Comment:
Chat.js doesn't have such option (that's why you need to accumulate yourself just above)